constraints/src/test/java/gov/nasa/jpl/aerie/constraints/time/SpansTest.java

```diff
@@ -0,0 +1,170 @@
+package gov.nasa.jpl.aerie.constraints.time;
+
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
```

Comment:
```suggestion
import java.util.List;
import java.util.stream.Stream;
```

IntelliJ is calling out some unused imports here

Reply:
removed